tests/foreman/ui/test_host.py

```diff
@@ -626,8 +632,31 @@ def test_negative_delete_primary_interface(session, module_host_template):
     :CaseLevel: System
     """
     interface_id = gen_string('alpha')
+    host_template = entities.Host(organization=module_org, location=module_location)
+    host_template.create_missing()
```

Comment:
This host entity that gets created just to get a domain ID and MAC address is never cleaned up.